concordium-sdk/src/main/java/com/concordium/sdk/responses/modulesource/ModuleSource.java

```diff
@@ -0,0 +1,19 @@
+package com.concordium.sdk.responses.modulesource;
+
+import concordium.ConcordiumP2PRpc;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Parsed response of
```

Comment:
This type of documentation is not really useful for the consumer of the SDK.

Reply:
done